research/object_detection/metrics/offline_eval_map_corloc.py

```diff
@@ -159,7 +159,7 @@ def main(argv):
       eval_config_path=FLAGS.eval_config_path)
 
   eval_config = configs['eval_config']
-  input_config = configs['eval_input_config']
+  input_config = configs['eval_input_configs']
```

Comment:
configs['eval_input_configs'] is a list so I think this should be configs['eval_input_configs'][0]

Reply:
I tried the changes proposed here ; 
https://github.com/tensorflow/models/issues/5411#issuecomment-483992817
It does the job